sdkproject/Assets/Mapbox/Unity/Map/DynamicZoomTileProvider.cs

```diff
@@ -92,7 +92,7 @@ void Update()
 			{
 				//already at highest level, don't do anything -> camera free to move closer
 				if (_dynamicZoomMap.Zoom == _dynamicZoomMap.MaxZoom) { return; }
-				_dynamicZoomMap.SetZoom(_dynamicZoomMap.Zoom + 1);
+				_dynamicZoomMap.SetZoomRange(_dynamicZoomMap.Zoom + 1);
```

Comment:
Just curious: Why the change from `SetZoom` to `SetZoomRange`?
This sets a single value, not a range of values - or am I missing something?

Reply:
You are correct - Changed Zoom to be a float value, and AbsoluteZoom to return an integer used for server requests.